eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -5,6 +5,7 @@ parameters:
   ResourceServiceDirectory: ""
   EnvVars: {}
   MaxParallel: 0
+  TimeoutInMinutes: 240
```

Comment:
Do we think we are at a point where we can lower the default for everyone?

Reply:
Perhaps invert this and make the default 60 and for any pipelines that need more time they set it in their pipeline?

Reply:
Good idea! Do we still want to merge JS pipelines so each service only has one single nightly test pipeline?  That might impact the default value we want to use here.

Reply:
cc @danieljurek 

Yes I do believe it is our goal to have one live test pipeline per service directory instead of per package.